src/main/java/com/denizenscript/denizencore/objects/core/ElementTag.java

```diff
@@ -2497,6 +2497,17 @@ else if (elem.equals("false")) {
         tagProcessor.registerStaticTag(MapTag.class, "parse_yaml", (attribute, object) -> {
             return (MapTag) CoreUtilities.objectToTagForm(YamlConfiguration.load(object.asString()).contents, attribute.context);
         });
+
+        // <--[tag]
+        // @attribute <ElementTag.millis_to_time>
+        // @returns TimeTag
+        // @group conversion
+        // @description
+        // Returns a TimeTag from the number of milliseconds since Jan 1, 1970.

```

Comment:
per discord -- should mention "epoch millis" / "unix time", and link to the tag(s) that return relevant values